cosign.py

```diff
@@ -16,48 +15,39 @@
 ci.log.configure_default_logging()
 logger = logging.getLogger(__name__)
 
-COSIGN_SIGNATURE_TYPE = "gardener.vnd/oci/cosign-signature"
```

Comment:
nit: maybe keep this constant and also reference it in `cosign_test.py`?

Reply:
We might. OTOH, duplicating the mimetype in unittest could actually considered to be better, as this will fail the test if the mimetype were to be changed unintentionally (typically, such values very rarely change)